src/main/java/org/cactoos/iterator/IteratorOf.java

```diff
@@ -23,64 +23,59 @@
  */
 package org.cactoos.iterator;
 
-import java.util.Collection;
```

Comment:
@Maratori Seems like you did some big refactoring in this class. Are you sure you merely renamed the class? Does it offer the same functionality?

Reply:
Yes it has the same functionality because both is just a simple Iterators.
Please have a look discussion about that (#641).